server/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -146,24 +169,15 @@ public Object get(String id)
           }
 
           @Override
-          public Map<String, LookupExtractorFactory> getAll()
+          public LookupsState getAll()
           {
-            return manager.getAll();
+            return manager.getAllLookupsState();
           }
 
           @Override
           public Object delete(String id)
           {
-            if (manager.get(id) == null) {
-              return null;
-            }
-            if (!manager.remove(id)) {
-              if (manager.get(id) == null) {
-                return null;
-              }
-              // We don't have more information at this point.
-              throw new RE("Could not remove lookup [%s]", id);
-            }
+            manager.remove(id);
```

Comment:
yes, .. delete() is kept only for backward compatibility with old coordinator code and to support rolling deployment.
we don't need it afterwards.. the lookup node api just need get and update